src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/cas2/Cas2ApplicationTest.kt

```diff
@@ -224,8 +224,8 @@ class Cas2ApplicationTest : IntegrationTestBase() {
         }
       }
 
-      fun unexpiredDateTime() = OffsetDateTime.now().randomDateTimeBefore()
```

Comment:
i don't suppose you fancy removing the default 'maxDays' attribute on randomDateTimeBefore(). I was confused by this change until i looked at the function definition and realised it used 14 by default, which i don't think is intuitive

Reply:
I've removed the default values from all the functions in RandomData.kt now.  Should make it more readable across the board.  Re-requested review again.

Reply:
That's fantastic, thank you!

Reply:
(pls. consider this re-reviewed 😄  )